server/routes.js

```diff
@@ -133,6 +133,9 @@ export default app => {
   app.param('paranoidtransactionid', params.paranoidtransactionid);
   app.param('expenseid', params.expenseid);
 
+  const isProduction = process.env.NODE_ENV === 'production';
+  const isStaging = process.env.NODE_ENV === 'staging';
+  const isDevelopment = !isProduction && !isStaging;
```

Comment:
What about `test` and `circlecli` environments. Maybe just check for `=== 'development'` there.

Reply:
yeah, I thought the same but that would break that current functionality for API V1 and maybe the workflow of another dev includes to check the playground during test. I don't know jajaja. But yeah, I can just check for `development`

Reply:
Done!